content/software-security/compliance/pci-dss-4/pci-dss-practices.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Overview of PCI DSS 4.0 Practices/Requirements"
+description: "Learn about the practices required for PCI DSS 4.0"
+lead: "Learn about the practices required for PCI DSS 4.0"
+type: "article"
+date: 2024-08-21T14:05:09+00:00
+lastmod: 2024-08-21T14:05:09+00:00
+contributors: []
+draft: false
+tags: ["compliance", "PCI DSS 4.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "pci-dss-4"
+weight: 003
+toc: true
+---
+
+PCI DSS 4.0, or PCI Data Security Standard is intended for all entities that store, process, or transmit cardholder data and/or authentication data that could impact the security of the cardholder data environment. This includes all entities interacting with information such as the following:
+
+| Cardholder Data | Authentication Data |
```

Comment:
This table threw me, because at first I thought there was a relationship between both columns. Maybe you could do two separate lists? Feel free to push back.

Reply:
I split the info into two more-easily-parsed tables. LMK what you think as I was waffling between this and simply making two unordered lists. I can go back and change this further.